### ISSUE TITLE
allow to debug without brk

### ISSUE THREAD

**Issue by luxferresum** [denali-js/denali#429]
It could be nice if denali could set `--debug` instead of `--inspect-brk` for action debugging.

**Comment by knownasilya** [comment]
The current one is necessary but the suggested one would also be nice to have.

**Comment by luxferresum** [comment]
I totally agree, we need both! But I would expect `--debug` to be without brk, and suggest `--debug-brk` instead

**Comment by knownasilya** [comment]
Good point.
